### ISSUE TITLE
Invalid token for access

### ISSUE THREAD

**Issue by ryanmboyce** [home-assistant/hassio#741]
I am receiving the following error when trying to run an add-on that worked just fine a few days ago and I didn't make any changes. 

```
- id: '165237682817201'
  alias: Backup Service
  trigger:
  - at: '15:10:00'
    platform: time
  action:
  - data:
      addon: ce20243c_remote_backup
    service: hassio.addon_start
```
Error on the supervisor:
`18-10-05 22:13:54 INFO (SyncWorker_11) [hassio.docker.interface] Clean fixated/remote-backup-armhf Docker application
18-10-05 22:13:56 INFO (SyncWorker_11) [hassio.docker.addon] Start Docker add-on fixated/remote-backup-armhf with version 0.3.0
18-10-05 22:13:56 WARNING (MainThread) [hassio.api.security] /snapshots/new/full no role for ce20243c_remote_backup
18-10-05 22:13:56 ERROR (MainThread) [hassio.api.security] Invalid token for access /snapshots/new/full`

Doesn't matter if I try running it manually, still doesn't work. Any ideas?

**Comment by pvizeli** [comment]
What version do you have of home-assistant?

**Comment by ryanmboyce** [comment]
Version: 0.79.3

**Comment by pvizeli** [comment]
Oh, yes sorry. You need set `hassio_rule: manager`. I think we should create a rule in future like `backup`

**Comment by ryanmboyce** [comment]
Where is that set at? In main configuration.yaml?

**Comment by pvizeli** [comment]
In the Add-on: "remote_backup"

**Comment by ryanmboyce** [comment]
I tried to set this variable in the Config for the add-on but it doesn't stick around.

**Comment by tjorim** [comment]
It's not in the user-config section of the add-on, but the add-on itself.
Update the 'Remote Backup' add-on to version 0.3.1, that should fix this.

**Comment by ryanmboyce** [comment]
Confirmed fixed after latest update (0.3.2)
thx team!
